pkg/resourcetopologyexporter/resourcetopologyexporter.go

```diff
@@ -207,8 +208,9 @@ func (rm *ResourceMonitor) Run(eventsChan <-chan PollTrigger) (<-chan nrtupdater
 					Zones: zones,
 				}
 				tsEnd := time.Now()
+				tsDiff := tsEnd.Sub(tsBegin)
 
-				log.Printf("read request received at %v completed in %v", tsBegin, tsEnd.Sub(tsBegin))
+				prometheus.UpdateOperationDelayMetric("podResScan", float64(tsDiff.Milliseconds()))
```

Comment:
please replace "podResScan" with `podresources_scan`

Reply:
we should probably add another label with the content of `pt.Timer` here.

Reply:
Actually, the value of the label should be the return value of https://github.com/k8stopologyawareschedwg/resource-topology-exporter/blob/master/pkg/nrtupdater/nrtupdater.go#L58

Reply:
on second thought, let's postpone this change